content/docs/reference/collections.md

```diff
@@ -70,3 +70,36 @@ $localization:
 ### categories
 
 The `categories` key contains a list of categories that documents inside the collection can fall into. A collection's categories can be used in conjunction with the `g.categories` template function to iterate over the documents in the collection, grouped by category.
+
+## Nested collections
+
+You can create collections inside other collections simply creating a nested directory containing a blueprint
+
+```bash
+├─ /content*                     # All your content.
+|  ├─ /books                     # The collection "books".
+|     ├─ /_blueprint.yaml*       # The blueprint for "books".
+|     ├─ /index.md               # A document belonging to "books".
+|     ├─ /contacts.md            # A document belonging to "books".
+|     └─ /how_to_use_grow        # The collection "how_to_use_grow".
+|        ├─ /_blueprint.yaml*    # The blueprint for "how_to_use_grow".
+|        ├─ /chapter01.md        # A document belonging to "how_to_use_grow".
+|        ├─ /chapter02.md        # A document belonging to "how_to_use_grow".
+|        ├─ /chapter03.md        # A document belonging to "how_to_use_grow".
+|     └─ /grow_style_guide       # The collection "grow_style_guide".
+|        ├─ /_blueprint.yaml*    # The blueprint for "grow_style_guide".
+|        ├─ /chapter01.md        # A document belonging to "grow_style_guide".
+|        ├─ /chapter02.md        # A document belonging to "grow_style_guide".
+```
+
+Here, `books` is a collection that contains documents, namely `index.md` and `contacts.md`. It also contains two nested collections, `books/how_to_use_grow` and `books/grow_style_guide`, each one with their own blueprint and content files.
+
+Inside the view used by `books/index.md` the variable `doc.collection.basename` will be `books`, while for `books/how_to_use_grow/chapter01.md` the value will be `books/how_to_use_grow`, and for `books/grow_style_guide/chapter01.md` it will be `books/grow_style_guide`.
+
+Anywhere in the pod you can refer to nested collections using their full basename, e.g.
+
+``` jinja
+{% for chapter in g.collection('books/how_to_use_grow).list_docs() %}
```

Comment:
Missing a `'` after the collection name.

Reply:
Fixed